apps/obs-web/src/layouts/Default.vue

```diff
@@ -77,7 +77,17 @@
           </q-item-section>
         </q-item>
 
-        <q-item :to="onlineStatus ? '/log-book-capture' : ''" exact :disabled="!onlineStatus">
+        <q-item v-if="isAuthorized(['enforcement'])" :to="onlineStatus ? '/ole-efp-management' : ''" exact :disabled="!onlineStatus">
+          <q-item-section avatar>
+            <q-icon name="table" />
+          </q-item-section>
+          <q-item-section>
+            <q-item-label>EFP Management</q-item-label>
+            <q-item-label caption>add or edit current EFPs in the declarations app</q-item-label>
+          </q-item-section>
+        </q-item>
+
+        <q-item v-if="!isAuthorized(['enforcement'])" :to="onlineStatus ? '/log-book-capture' : ''" exact :disabled="!onlineStatus">
```

Comment:
I'd propose rewriting as:
<q-item v-if="isAuthorized(['captain', 'staff', 'debriefer'])...
so we're actively declaring who DOES have access to this link, as opposed to who shouldn't see it.

Reply:
Ahh, yeah, that thought crossed my mind.  So any of these roles that shouldn't be on these pages, not sure what the different roles are all for. Or is really just those three roles you had that I need? (Thanks for the review!)

0: "observer"
1: "debriefer"
2: "captain"
3: "permit_owner"
4: "provider"
5: "analyst"
6: "lab_analyst"
7: "data_steward"
8: "enforcement"
9: "vessel_owner"
10: "program_manager"
11: "gear_technician"
12: "lead_observer"
13: "logistical_admin"
14: "coordinator"
15: "trainer"
16: "reports_user"
17: "staff"
18: "development_staff"

Reply:
we're currently just using the 3 I listed.  In the future, I could see more roles using the logbook capture (as roles get better defined), but let's just go with the 3 for now.